+dj/Table.m

```diff
@@ -493,8 +466,8 @@ function syncDef(self)
                 fprintf('File %s.m is not found\n', self.className);
             else
                 if ~dj.set('suppressPrompt') ...
-                        && ~strcmpi('yes', dj.ask(sprintf('Update table declaration in %s?',path)))
-                    disp 'No? Table declaration left untouched.'
+                        && ~strcmpi('yes', dj.ask(sprintf('Update table definition in %s?',path)))
```

Comment:
Include explicit message stating that the table will be subclassed from the correct UserRelation - this will help point out the new syntax.